packages/webview_flutter/README.md

```diff
@@ -0,0 +1,63 @@
+# webview_flutter_tizen
+
+The Tizen implementation of [`webview_flutter`](https://github.com/flutter/plugins/tree/master/packages/webview_flutter).
+
+## Supported devices
+
+This plugin is available on these types of devices:
+
+- Galaxy Watch or TV (running Tizen 5.5 or later)
+
+## Required privileges
+
+To use this plugin, add below lines under the `<manifest>` section in your `tizen-manifest.xml` file,
+
+```xml
+<privileges>
+  <privilege>http://tizen.org/privilege/internet</privilege>
+</privileges>
+
+## Usage
+
+This package is not an _endorsed_ implementation of `webview_flutter`. Therefore, you have to include `webview_flutter_tizen` alongside `webview_flutter` as dependencies in your `pubspec.yaml` file.
+
+```yaml
+dependencies:
+  webview_flutter: ^1.0.6
+  webview_flutter_tizen: ^1.0.0
```

Comment:
- Broken markdown
- The current version is 0.1.0

Reply:
Done.